index.html

```diff
@@ -762,18 +839,137 @@ <h2><dfn>CaptureController</dfn></h2>
             [Exposed=Window, SecureContext]
             interface CaptureController {
               constructor();
-              // TODO: Add setFocusBehavior() in a separate PR.
+              undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
           <dl data-link-for="CaptureController" data-dfn-for="CaptureController" class="methods">
             <dt>
               <dfn>constructor</dfn>
             </dt>
+            <dd>
+              Run the following steps:
+              <ol>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\isBound]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot tracks whether [=this=] has been associated with a [=capture-session=].
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\videoTrack]]</dfn>,
+                    initialized to <code>null</code>.
+                    <div class="note">
+                      <!-- TODO: Ensure surfaceSwitching closes the focus window. -->
+                      This internal slot tracks with which [=display surface=] the [=capture-session=] is associated.
+                      It is distinct from {{CaptureController/[[isBound]]}} in that {{CaptureController/[[isBound]]}}
+                      is set even if {{MediaDevices/getDisplayMedia()}} rejects.
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\initialFocusLost]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot ensures that calls to {{CaptureController/setFocusBehavior()}} fail
+                      without throwing if the user manually focuses another tab or native application.
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\focusWindowClosed]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot tracks explicit closure of the window of opportunity, which happens when
+                      {{CaptureController/setFocusBehavior()}} is called, or the task to close the window executes.
+                      As applications can easily avoid such errors, {{CaptureController/setFocusBehavior()}} when
+                      this slot is set to <code>true</code> throw.
```

Comment:
I found it hard to read.

```suggestion
                      As applications can easily avoid such errors, {{CaptureController/setFocusBehavior()}} throws 
                      when this slot is set to <code>true</code>.
```